bittide-shake/src/Clash/Shake/Cargo.hs

```diff
@@ -0,0 +1,31 @@
+-- SPDX-FileCopyrightText: 2023 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+module Clash.Shake.Cargo where
+
+import Prelude
+
+import System.Exit
+import System.IO
+import System.Process
+
+cargoBuildFirmwareProgram ::
+  FilePath ->
+  -- ^ Directory to perform the call in
+  Bool ->
+  -- ^ release mode?
+  IO ()
+cargoBuildFirmwareProgram workingDir release = do
+  let
+    createProc =
+      (proc "cargo" (["build"] <> ["--release" | release]))
```

Comment:
```suggestion
      (proc "cargo" ("build" : ["--release" | release]))
```

Reply:
Also, don't we want to use `cargo.sh` @cuddlefishie ?

Reply:
That script is mostly for users who want to check/build their code in the project root, since this one takes a working directory there's no need to invoke that script